### PR TITLE
update to recent header changes and quic-transport restructure

### LINKED ISSUE #57: update -manageability for changes to packet headers
Review the text of the -manageability draft to ensure that changes to headers in the latest version of -transport are correctly tracked.

### PR DESCRIPTION
**Pull request by britram**
Fix #57.

### CODE REVIEW

File: draft-ietf-quic-manageability.md [hunk 1]
```diff
@@ -124,27 +124,29 @@ QUIC packets may have either a long header, or a short header. The first bit
...
 
-Short headers are used after connection establishment. The only information
-they contain for inspection on the path is an optional, variable-length
-destination connection ID.
+Short headers are used after connection establishment, and contain only an
+optional destination connection ID and the spin bit for RTT measurement.
 
-As of draft version 13 of the QUIC transport document, the following
-information may be exposed in QUIC packet headers:
+As of draft version 18 of the QUIC transport document, the following
```
Comment (martinthomson) [inline draft-ietf-quic-manageability.md]: Is this style of statement something you want to keep?  You could just say:

> The following information is exposed is QUIC packet headers:

And allow for the possibility of some skew.  That seems increasingly unlikely now though.

File: draft-ietf-quic-manageability.md [hunk 2]
```diff
@@ -124,27 +124,29 @@ QUIC packets may have either a long header, or a short header. The first bit
...
+information is exposed in QUIC packet headers:
 
-- header type: the long header has a 7-bit packet type field following the
+- For demultiplexing purposes, the second most significant bit of
+  the first octet every QUIC packet of the current version is set to 1.
```
Comment (martinthomson) [inline draft-ietf-quic-manageability.md]: list construction isn't parallel here

File: draft-ietf-quic-manageability.md [hunk 3]
```diff
@@ -160,17 +162,24 @@ information may be exposed in QUIC packet headers:
...
 
 - length: the length of the remaining quic packet after the length field,
-  present on
-  long headers. This field is used to implement coalesced packets during the
-  handshake (see {{coalesced}}).
+  present on long headers. This field is used to implement coalesced packets
+  during the handshake (see {{coalesced}}).
+
+- spin bit: the spin bit supports passive measurement of RTT, and is present on
+  QUIC short packet headers. See {{sec-rtt}}.
```
Comment (martinthomson) [inline draft-ietf-quic-manageability.md]: add the token field (with accompanying length) from the Initial packet

File: draft-ietf-quic-manageability.md [hunk 4]
```diff
@@ -160,17 +162,24 @@ information may be exposed in QUIC packet headers:
...
+  is encrypted, and therefore not of use to on-path observers. The offset of the
+  packet number is encoded in the header for packets with long headers, while it
+  is implicit (depending on Destination Connection ID length) in short header
+  packets. The length of the packet number is cryptographically obfuscated.
 
-- key phase: The Key Phase bit, present in short headers identifies the key
-  used to encrypt the packet during key rotation.
+- key phase: The Key Phase bit, present in short headers, specifies the keys
+  used to encrypt the packet, supporting key rotation. The Key Phase bit is
+  cryptographically obfuscated.
```
Comment (martinthomson) [inline draft-ietf-quic-manageability.md]: Separately observe that Retry and Version Negotiation packets are not encrypted or obfuscated in any way.

File: draft-ietf-quic-manageability.md (hunk 1 above)
Comment (britram) [reply draft-ietf-quic-manageability.md]: done, ignoring skew possibility...

File: draft-ietf-quic-manageability.md (hunk 2 above)
Comment (britram) [reply draft-ietf-quic-manageability.md]: yeah that annoyed me as i typed it. fixed.

File: draft-ietf-quic-manageability.md (hunk 3 above)
Comment (britram) [reply draft-ietf-quic-manageability.md]: done.

File: draft-ietf-quic-manageability.md (hunk 4 above)
Comment (britram) [reply draft-ietf-quic-manageability.md]: also done

**Pull request merged**